test/test_script.py

```diff
@@ -384,3 +388,48 @@ def test_cmd_unset_multi(self):
         self.parser.eval("$unset(performer:*)", context)
         self.assertNotIn('performer:bar', context)
         self.assertNotIn('performer:foo', context)
+
+    def test_cmd_inmulti(self):
+        context = Metadata()
+        context["foo"] = "First; Second; Third"
+        self.assertEqual(
+            self.parser.eval("$in(%foo%,Second)", context), "1")
+        self.assertEqual(
+            self.parser.eval("$in(%foo%,irst; Second; Thi)", context), "1")
+        self.assertEqual(
+            self.parser.eval("$in(%foo%,First; Second; Third)", context), "1")
+        self.assertEqual(
+            self.parser.eval("$inmulti(%foo%,Second)", context), "")
+        self.assertEqual(
+            self.parser.eval("$inmulti(%foo%,irst; Second; Thi)", context), "")
+        self.assertEqual(
+            self.parser.eval("$inmulti(%foo%,First; Second; Third)", context), "1")
+
+        context["foo"] = ["First", "Second", "Third"]
+        self.assertEqual(
+            self.parser.eval("$in(%foo%,Second)", context), "1")
+        self.assertEqual(
+            self.parser.eval("$in(%foo%,irst; Second; Thi)", context), "1")
+        self.assertEqual(
+            self.parser.eval("$in(%foo%,First; Second; Third)", context), "1")
+        self.assertEqual(
+            self.parser.eval("$inmulti(%foo%,Second)", context), "1")
```

Comment:
Please add a test with a user-defined separator. At the moment, that doesn't appear to work:

```
======================================================================
ERROR: test_cmd_inmulti (test.test_script.ScriptParserTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/wieland/dev/picard/test/test_script.py", line 416, in test_cmd_inmulti
    self.parser.eval("$inmulti(%foo%,Second, :)", context), "1")
  File "/home/wieland/dev/picard/picard/script.py", line 284, in eval
    return ScriptParser._cache[key].eval(self)
  File "/home/wieland/dev/picard/picard/script.py", line 122, in eval
    result.append(item.eval(state))
  File "/home/wieland/dev/picard/picard/script.py", line 114, in eval
    return function(parser, *args)
  File "/home/wieland/dev/picard/picard/script.py", line 421, in func_inmulti
    return func_in(parser, _get_multi_values(parser, haystack, separator), needle)
  File "/home/wieland/dev/picard/picard/script.py", line 340, in _get_multi_values
    return multi.split(separator) if separator else [multi]
TypeError: must be str or None, not ScriptExpression

----------------------------------------------------------------------
Ran 61 tests in 0.033s

FAILED (errors=1)
```